extendeddatacrossword.go

```diff
@@ -88,15 +95,13 @@ func RepairExtendedDataSquare(
 
 	err = eds.prerepairSanityCheck(rowRoots, colRoots, bitMat, codec)
 	if err != nil {
+		if isByzantineError(err) {
+			return eds, err
+		}
 		return nil, err
 	}
 
-	err = eds.solveCrossword(rowRoots, colRoots, bitMat, codec)
-	if err != nil {
-		return nil, err
-	}
-
-	return eds, err
+	return eds, eds.solveCrossword(rowRoots, colRoots, bitMat, codec)
```

Comment:
Can you add a test to show that the `eds` returned is actually the correct one here (i.e. the one _after_ `solveCrossword`)?

Reply:
done